modules/manage/pages/schema-registry.adoc

```diff
@@ -736,6 +798,15 @@ res = requests.delete(f'{base_uri}/subjects/sensor-value/versions/1', params=pay
 pretty(res)
 ```
 
+--
+rpk::
```

Comment:
The [docs](https://docs.redpanda.com/current/manage/schema-registry/#hard-delete-a-schema) currently mention that to perform a hard delete, the API endpoint has to be called twice, with the second call including the "permanent" parameter. Does that also apply to the rpk command or is running the command once sufficient for a hard delete?

Reply:
In `rpk` we have a `--permanent` flag that does the 2 deletes for you:

```
rpk registry schema-delete <subject> --schema-version latest --permanent```

Same for `subject`